elastalert/kibana.py

```diff
@@ -166,6 +166,8 @@
                   u'style': u'dark',
                   u'title': u'ElastAlert Alert Dashboard'}
 
+kibana4_time_temp = "(refreshInterval:(display:Off,section:0,value:0),time:(from:'%s',mode:absolute,to:'%s'))"
```

Comment:
I don't think Kibana4 will work if there are spaces and I don't think it's worth it to add spaces only to strip them out immediately.